### PR TITLE
Hawkular APM 0.14.0.Final with service comparison UI

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@jpkrohling @pavolloffay @theute @pilhuhn Comments welcome.

**Comment by theute** [comment]
Image is not showing up

**Comment by objectiser** [comment]
@theute Sorry about that, changed the date and forgot to update the image link.

**Comment by jpkrohling** [comment]
It seems the image is still not showing up, but other than that, LGTM

**Comment by jpkrohling** [comment]
Sorry, false alarm, it *is* showing up now.

File: src/main/jbake/content/blog/2017/02/04/hawkular-apm-service-deployments.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+the details are automatically obtained from the environment - however when running services outside of OpenShift, these values can be provided
+as environment variables (_HAWKULAR_APM_SERVICE_NAME_ and _HAWKULAR_APM_BUILDSTAMP_ respectively).
+
+At the top of the new page, you can select the service name and then a particular version (or _All_) to see the performance of that service/version
+over the defined time span. The number of transactions and faults are also shown. Once the 'Add' button is pressed, the details will be displayed.
+
+To perform a comparison with either the same service, but different version, or between different services - simply select their details and press
+'Add' again. The combined information will be shown in the same charts, allowing direct comparison of their information.
+
+The following image shows the deployment of three versions of the Account Manager service from the OpenTracing vertx example.
```
Comment (pavolloffay) [inline src/main/jbake/content/blog/2017/02/04/hawkular-apm-service-deployments.adoc]: It would be good to add a link to the code.

**Comment by pavolloffay** [comment]
LGTM :+1:

File: src/main/jbake/content/blog/2017/02/04/hawkular-apm-service-deployments.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, opentracing, openshift
+
+With the release of https://github.com/hawkular/hawkular-apm/releases/tag/0.14.0.Final[0.14.0.Final on Hawkular APM] we have introduced a new feature in support of continuous deployment of microservices.
+When a new version of a service is deployed, potentially as part of a deployment strategy (such as Canary, A/B, Blue/Green), it is important
+to be able to assess the impact the new version of the service has in comparison to existing/previous versions.
+
+The new "Services" tab in the UI provides just such a feature. When trace data is reported by the Hawkular APM OpenTracing provider (Java
+or JavaScript), it identifies the service name and optional version (known as a buildStamp). When the services are deployed within OpenShift
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2017/02/04/hawkular-apm-service-deployments.adoc:13]: I would not put the "When trace data ... " part here, as everything else is about the UI. This how-to-identify is more like a footnote to me.
Why not put that in a section after the chart?

**Comment by objectiser** [comment]
@pilhuhn Thanks

**Pull request merged**